### ISSUE TITLE
[BUG]: alt-text checking bot to rerun

### ISSUE THREAD

**Issue by bsipocz** [the-turing-way/the-turing-way#4347]
### Description

It looks like the alt-text checking bot comments on missing alt-text, but then it's not triggered again to revise its comment once edits are made.

See https://github.com/the-turing-way/the-turing-way/issues/4346#issuecomment-3255164407

Would be nice to retrigger it so we don't keep outdated info on the issue.

### Changes required

Update the version of the alt text bot in `accessibility-alt-text-bot.yml` to at least 1.6.0

**Comment by JimMadge** [comment]
@TheAbstract you added this workflow. Is there anything you think can be done here?

**Comment by JimMadge** [comment]
The workflow is triggered when issues (and so on) are edited. I'm assuming the bot _only_ posts when there is a problem, and otherwise does nothing.

So I'm guessing that what happens is,

- item is created/edited with missing alt-text: bot posts message about missing alt-text
- item is edited to add alt-text: bot identifies no problems, does nothing

I suspect improving this might require updating [the action](https://github.com/github/accessibility-alt-text-bot). Maybe adding a condition to check if there is a previous message from the bot about missing alt-text?

**Comment by JimMadge** [comment]
Oh, actually, looks like this feature was added in [v1.6.0](https://github.com/github/accessibility-alt-text-bot/releases/tag/v1.6.0)

Action here is to update the version of the bot. (I'm not sure why dependabot doesn't seem to be catching this, perhaps because a specific minor version is specified.)

**Comment by bsipocz** [comment]
> Action here is to update the version of the bot. (I'm not sure why dependabot doesn't seem to be catching this, perhaps because a specific minor version is specified.)

Yes, I suppose we need to get rid of the minor version ignore conditional from the dependabot config. I mean it runs monthly, it should really pick up those changes on that timescale as in my experience folks hardly every change the major version

**Comment by JimMadge** [comment]
> Yes, I suppose we need to get rid of the minor version ignore conditional from the dependabot config. I mean it runs monthly, it should really pick up those changes on that timescale as in my experience folks hardly every change the major version

I meant _patch version_ 🤦.
The others get updated, so I suspect it is because here we have specified `major.minor.patch`.

You're right that the ignore minor versions isn't really doing anything for us here. The monthly schedule already reduces spam.

Maybe the real question there is, do we want to specify `major.minor.patch` where possible in the workflow files. That would have the advantage that we know exactly which version of an action was used (ignoring for the moment that you can move tags 😅). We could only specify `major` or `major.minor` and we would always get the latest matching release.

**Comment by da5nsy** [comment]
<img width="267" height="268" alt="This is a random image just for testing purposes which I found on my desktop" src="https://github.com/user-attachments/assets/f5fd8afc-4986-4048-a6ac-5ed3b6c024fb" />

Testing with a random image

**Comment by da5nsy** [comment]
(Writing here because the test was so successful that the result is no longer visible! 🎉 The bot sent a message, and then deleted it once I had updated the comment to add alt text 🎉 )
